src/apps/core/views/svg-container.html

```diff
@@ -468,557 +468,703 @@
 		<!-- | ============================ -->
 		<!-- |
 			 |  [1] - Account tree
```

Comment:
This numbered list is a nice addition but I believe this is not included in the library.

Updating telicon to a new version should be fairly simple and one should not have to regenerate that list every time. Plus, there is already a way lookup existing icons from the github page.

Reply:
That is true, it takes too much time to update the list and the numbering for each symbol. I'll remove them.